GiniVision/Classes/CameraViewController.swift

```diff
@@ -131,13 +143,15 @@ public typealias CameraErrorBlock = (_ error: CameraError) -> ()
         if let validCamera = camera {
             cameraState = .valid
             previewView.session = validCamera.session
-            (previewView.layer as! AVCaptureVideoPreviewLayer).videoGravity = AVLayerVideoGravityResizeAspect
+            (previewView.layer as! AVCaptureVideoPreviewLayer).videoGravity = AVLayerVideoGravityResizeAspectFill
+            (previewView.layer as! AVCaptureVideoPreviewLayer).connection?.videoOrientation = UIDevice.current.userInterfaceIdiom == .pad ?
+                orientationsMapping[UIDevice.current.orientation] ?? .portrait :
+                .portrait
```

Comment:
I prefer clarity over brevity, because imho the time you save writing this someone will spend in the future and with interest.

I recommend using a method to set the video orientation where the logic is made easy to read with if conditional.

I like the elvis operator, but how can it happen that the orientationsMapping does not have a corresponding item for the device orientation?